packages/babel-plugin-fbtee/src/index.tsx

```diff
@@ -149,9 +149,8 @@ export type Phrase = FbtCallSiteOptions & {
   line_end: number;
   project: string;
 } & ObjectWithJSFBT;
-type ChildToParentMap = {
-  [childIndex: number]: number;
-};
+
+type ChildToParentMap = Map<number, number>;
```

Comment:
This was one of the cleanups that stops converting numbers to strings and back for the collection code.